src/Tribe/Repositories/Ticket/RSVP.php

```diff
@@ -7,6 +21,45 @@
  */
 class Tribe__Tickets__Repositories__Ticket__RSVP extends Tribe__Tickets__Ticket_Repository {
 
+	use Get_Field;
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$post_type = ( $this->ticket_types() )['rsvp'] ?? 'tribe_rsvp_tickets';
+
+		$this->create_args['post_type'] = $post_type;
+
+		$this->default_args = [
+			'post_type' => $post_type,
+			'orderby'   => [ 'date', 'ID' ],
+		];
+
+		// Add RSVP-specific field aliases.
+		$this->update_fields_aliases = array_merge(
+			$this->update_fields_aliases,
+			[
+				'event_id'          => '_tribe_rsvp_for_event',
+				'price'             => '_price',
+				'stock'             => '_stock',
+				'sales'             => 'total_sales',
+				'manage_stock'      => '_manage_stock',
+				'start_date'        => '_ticket_start_date',
+				'end_date'          => '_ticket_end_date',
+				'show_description'  => '_tribe_ticket_show_description',
+				'show_not_going'    => '_tribe_rsvp_show_not_going',
+				'capacity'          => '_tribe_ticket_capacity',
+				'global_stock_mode' => '_global_stock_mode',
+				'global_stock_cap'  => '_global_stock_cap',
```

Comment:
I'm pretty sure most of those keys are constants in `TEC\Tickets\Commerce\Ticket`. Some of them may also be part of `Tribe__Tickets__Tickets_Handler`. I would suggest to reuse the constants instead of hard-coding them here.